packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/TransformController.java

```diff
@@ -31,28 +31,6 @@ public class TransformController {
 	final SimulationServiceProxy simulationServiceProxy;
 	final Messages messages;
 
-	@PostMapping("/mathml-to-acset")
```

Comment:
The member variable `skemaProxy` can also be removed

Reply:
ahh yes done!